exchanges/btcmarkets/btcmarkets_wrapper.go

```diff
@@ -1056,3 +1061,50 @@ func (b *BTCMarkets) GetHistoricCandlesExtended(ctx context.Context, p currency.
 	ret.SortCandlesByTimestamp(false)
 	return ret, nil
 }
+
+// UpdateCommissionFees updates current fees associated with account
+func (b *BTCMarkets) UpdateCommissionFees(ctx context.Context, a asset.Item) error {
+	if a != asset.Spot {
+		return fmt.Errorf("%s %w", a, asset.ErrNotSupported)
+	}
+
+	temp, err := b.GetTradingFees(ctx)
+	if err != nil {
+		return err
+	}
+
+	for x := range temp.FeeByMarkets {
+		err = b.Fees.LoadDynamicFeeRate(temp.FeeByMarkets[x].MakerFeeRate,
+			temp.FeeByMarkets[x].TakerFeeRate,
+			a,
+			temp.FeeByMarkets[x].MarketID)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+// UpdateTransferFees updates transfer fees for cryptocurrency withdrawal and
+// deposits for this exchange
+func (b *BTCMarkets) UpdateTransferFees(ctx context.Context) error {
+	temp, err := b.GetWithdrawalFees(ctx)
+	if err != nil {
+		return err
+	}
+
+	var transferFees []fee.Transfer
+	for x := range temp {
+		if temp[x].AssetName.Item == currency.AUD.Item {
+			// Filter out fiat bank transfer
+			continue
+		}
```

Comment:
Shouldn't we include the bank transfer fee? Just for the fun event that btc markets does actually change it dynamically

Reply:
Yeah, good point, currently it's free, but that could change.